### PR TITLE
raft: remove check quorum feature

### PR DESCRIPTION
**Pull request by NingLin-P**
Signed-off-by: linning <linningde25@gmail.com>

### CODE REVIEW

**Comment by Connor1996** [comment]
Please fix the conflict

**Comment by NingLin-P** [comment]
> Please fix the conflict

Done.

File: raft/raft.go [hunk 1]
```diff
@@ -797,53 +760,15 @@ func (r *Raft) Step(m pb.Message) error {
...
-			// with "pb.MessageType_MsgAppendResponse" of higher term would force leader to step down.
-			// However, this disruption is inevitable to free this stuck node with
-			// fresh election.
-			r.send(pb.Message{To: m.From, MsgType: pb.MessageType_MsgAppendResponse})
-		} else {
-			// ignore other cases
-			r.logger.Infof("%x [term: %d] ignored a %s message with lower term from %x [term: %d]",
-				r.id, r.Term, m.MsgType, m.From, m.Term)
-		}
+		r.send(pb.Message{To: m.From, MsgType: pb.MessageType_MsgAppendResponse})
```
Comment (Connor1996) [inline raft/raft.go]: seems don't need to send it?

File: raft/raft.go [hunk 2]
```diff
@@ -797,53 +760,15 @@ func (r *Raft) Step(m pb.Message) error {
...
-			// When follower gets isolated, it soon starts an election ending
-			// up with a higher term than leader, although it won't receive enough
-			// votes to win the election. When it regains connectivity, this response
-			// with "pb.MessageType_MsgAppendResponse" of higher term would force leader to step down.
-			// However, this disruption is inevitable to free this stuck node with
-			// fresh election.
-			r.send(pb.Message{To: m.From, MsgType: pb.MessageType_MsgAppendResponse})
-		} else {
-			// ignore other cases
-			r.logger.Infof("%x [term: %d] ignored a %s message with lower term from %x [term: %d]",
```
Comment (Connor1996) [inline raft/raft.go]: Please print the log

**Pull request merged**